server/src/test/java/org/kaazing/gateway/server/context/resolve/DefaultClusterContextTest.java

```diff
@@ -407,58 +409,33 @@ private void addToClusterState(CollectionsFactory factory, MemberId memberId, in
         memberIdBalancerMap.put(memberId, myBalanceTargets);
     }
 
-    private void validateClusterState(ClusterContext... clusterContext) {
-        Set<String> balanceTargets = new HashSet<>();
-        if (clusterContext[0] != null) {
-            balanceTargets.add("ws://node1.example.com:8080/path");
-        }
-        if (clusterContext[1] != null) {
-            balanceTargets.add("ws://node2.example.com:8080/path");
-        }
-        balanceTargets.add("ws://node3.example.com:8080/path");
-        balanceTargets.add("ws://node4.example.com:8080/path");
-        if (clusterContext[0] != null) {
-            if (!validateSharedBalancer(clusterContext[0].getCollectionsFactory(), balanceTargets)) {
-                fail("Expected 4 URIs as balance targets in cluster member 1's memory");
-            } else {
-                List<String> myBalanceTargets = new ArrayList<>();
-                myBalanceTargets.add("ws://node1.example.com:8080/path");
-                if (!validateMemberIdBalancerMap(clusterContext[0].getCollectionsFactory(), clusterContext[0]
-                        .getLocalMember(), myBalanceTargets)) {
-                    fail("Expected ws://node1.example.com:8080/path as balance target provided by cluster member 1");
-                }
-            }
+    private void validateClusterState(List<ClusterContext> activeMembers, Map<ClusterContext, String> clusterContextToServiceUrlMap) {
```

Comment:
activeMembers should be renamed to expectedActiveMembers and clusterContextToServiceUrlMap should be renamed to expectedClusterContextToServiceUrlMap